docs/deploying/deploy-smart-contracts.mdx

```diff
@@ -18,29 +18,38 @@ Example: To deploy the contract to the Sepolia network, run the command below:
 yarn deploy --network sepolia
 ```
 
-2. Generate a new account or add one to deploy the contract(s) from. Additionally you will need to add your Alchemy API key. Rename `.env.example` to `.env` and fill the required keys.
-
-```
-ALCHEMY_API_KEY="",
-DEPLOYER_PRIVATE_KEY=""
-```
+## 2. Generate a new account or add one to deploy the contract(s) from.
 
 The deployer account is the account that will deploy your contracts. Additionally, the deployer account will be used to execute any function calls that are part of your deployment script.
 
 You can generate a random account / private key with `yarn generate` or add the private key of your crypto wallet. `yarn generate` will create a random account and add the DEPLOYER_PRIVATE_KEY to the .env file. You can check the generated account with `yarn account`.
 
-3. Deploy your smart contract(s)
+## 3. Deploy your smart contract(s)
 
 Run the command below to deploy the smart contract to the target network. Make sure to have some funds in your deployer account to pay for the transaction.
 
 ```
 yarn deploy --network network_name
 ```
 
-4. Verify your smart contract
+## 4. Verify your smart contract
 
 You can verify your smart contract on Etherscan by running:
 
 ```
 yarn verify --network network_name
 ```
+
+## Configuration of Third-Party Services for Production-Grade Apps.
+
+By default, Scaffold-ETH 2 provides predefined API keys for popular services such as Alchemy and Etherscan. This allows you to begin developing and testing your applications more easily, avoiding the need to register for these services.
+
+For production-grade applications, it's recommended to obtain your own API keys (to prevent rate limiting issues). You can configure these at:
+
+- `ALCHEMY_API_KEY` variable in `packages/hardhat/.env` and `packages/nextjs/.env.local`. You can create API keys from the [Alchemy dashboard](https://dashboard.alchemy.com/).
```

Comment:
In case, we go with https://github.com/scaffold-eth/se2-docs/pull/10#issuecomment-1666826061

We can make this copy pastable too? 
`ALCHEMY_API_KEY` variable in `packages/hardhat/.env` and `packages/nextjs/.env.local`.

```
ALCHEMY_API_KEY=Your-ALCHEMY-API-KEY
```

But the current one also looks good !!!

Reply:
I'm afraid that if we set all variables in a copy-pastable code snippet the content looks less readable, maybe not in this case but then I guess we should do it in all the places of docs where we set variable names.

We could set all the commands in copy-pastable code snippets and leave the variables like that, what do you think?

I'll send an example of how could look

Reply:
> I'm afraid that if we set all variables in a copy-pastable code snippet the content looks less readable, 

> We could set all the commands in copy-pastable code snippets and leave the variables like that, 

Yes, Pablo makes sense !! Let's keep only the commands copy-pastable 🙌